dts_test_project/dts_test_project/settings.py

```diff
@@ -122,3 +121,34 @@
 # https://docs.djangoproject.com/en/1.6/howto/static-files/
 
 STATIC_URL = '/static/'
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'filters': {
+        'require_debug_false': {
+            '()': 'django.utils.log.RequireDebugFalse'
+        },
+    },
+    'formatters': {
+        'simple': {
+            'format': '%(levelname)-7s %(asctime)s %(message)s',
+        },
+    },
+    'handlers': {
+        'null': {
+            'class': 'logging.NullHandler',
+        },
+        'console': {
+            'class': 'logging.StreamHandler',
+            'formatter': 'simple',
+        },
+    },
+    'loggers': {
+        '': {
+            'handlers': ['null'],
```

Comment:
This wasn't strictly necessary, but it does make life easier to be able to quickly switch `console` for `null` and get all the SQL interactions displayed.